src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -627,6 +627,13 @@ paths:
     post:
       operationId: ValidatorsUptime
       summary: Get Validators Uptime
+      description: |
+        Returns validators uptime data for time range limited by from_state_version and at_state_version.
+        `at_state_version` defines upper boundary (inclusive) for queried data. i.e `{ at_state_version = {epoch = 10} }`, will return uptime statistics from genesis till 10.
```

Comment:
Shouldn't we document those properties directly in `ValidatorsUptimeRequest` definition? Properties "belong" to the request definition not endpoint definition in my opinion.

Endpoint docs:
Returns validator uptime for selected time range.

Request docs:
Returns... bla bla bla... 
from_state_version - lower time range
at_statE_version - upper time range
and so on and so forth